pontoon/settings/base.py

```diff
@@ -219,10 +221,12 @@ def path(*args):
 ANONYMOUS_USER_ID = -1
 GUARDIAN_RAISE_403 = True
 
-PIPELINE_COMPILERS = (
+__PIPELINE_COMPILERS = (
```

Comment:
Is this a fix of a previous bug that went undetected, or are you kind of commenting that key out? It's weird that we have keys starting with underscores, is that a thing that some libraries use?

Reply:
it was me commenting it out - i left it there because i wasnt sure - but now i think we can safely remove